lib/routes/dtfb.js

```diff
@@ -1,35 +1,79 @@
 const router = require('express').Router();
 const fetch = require('node-fetch');
 
+async function getTournamentDates() {
+  const result = await fetch('https://dtfb.de/index.php/termine');
+  const html = await result.text();
+  var reg = new RegExp(/(3[01]|[12][0-9]|0?[1-9])\.(1[012]|0?[1-9])\.\s/g);
```

Comment:
you should use `let` for variables that change. `var` is old javascript 😉

Reply:
Right. I always mixup those two 🙈